academic-services/knowledge-exploration-service/resources-pricing.md

```diff
@@ -4,15 +4,15 @@ description: Provides Azure cost estimator links to model the cost of using Micr
 ms.topic: reference
 ms.date: 2020-02-10
 ---
-# Microsoft Academic Knowledge Exploration Service pricing
+# Microsoft Academic Knowledge Exploration Service Pricing
 
-Below are suggested hardware SKU's for various MAKES usage scenarios. These are only *suggested configurations*; you should experiment and determine the most appropriate configuration that best fits your needs.
+MAKES can be hosted on any virtual machine SKU that runs Windows OS. Below are suggested hardware SKU's for various MAKES usage scenarios. These are only *suggested configurations*; you should experiment and determine the most appropriate configuration that best fits your needs.
 
 API Purpose | Description | Host Machine Sku | Cost estimate
 --- | --- | --- | ---
-Sub-graph access | For enterprise or domain specific solutions that only require a smaller sub-graph of the full Microsoft Academic Graph (MAG). | Standard_DS4_v3  | [Pricing calculator](https://azure.com/e/c04290edd8bf4db08ba23db8392430c4)
-Full graph access | Provides moderately fast, responsive API method response times over the entirety of the Microsoft Academic Graph (MAG). |  Standard_DS14_v2 | [Pricing calculator](https://azure.com/e/9bfe795705424b26a118639a198adfbd)
-Fast, full graph access | For production scenarios that require very fast, responsive API method response times over the entirety of the Microsoft Academic Graph (MAG).  | Standard_E32s_v3 |[Pricing calculator](https://azure.com/e/2c1ee1b19db84b7dbb6d3eb7b5492d6c)
+Sub-graph access | For enterprise or domain specific solutions that only require a smaller sub-graph of the full Microsoft Academic Graph (MAG). | Standard_DS4_v3  | [Pricing calculator](https://azure.com/e/4a67bc36bc864ed5ac73d4c4724f0886)
```

Comment:
>Standard_DS4_v3 [](start = 147, length = 15)

Shouldn't that be Standard_D4_v3? I thought the DS series were the ones with smaller temp storage